packages/terra-application-layout/src/menu/_ApplicationMenu.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import AppDelegate from 'terra-app-delegate';
+import ApplicationMenuLayout from 'terra-application-menu-layout';
+import { ApplicationMenuName } from 'terra-application-name';
+import RoutingStackDelegate from 'terra-navigation-layout/lib/RoutingStackDelegate';
+import { processedRoutesPropType } from 'terra-navigation-layout/lib/configurationPropTypes';
+import { ApplicationMenuUtility } from 'terra-application-utility';
+import { disclosureType as modalDisclosureType } from 'terra-modal-manager';
+import { availableDisclosureSizes } from 'terra-disclosure-manager';
+
+import 'terra-base/lib/baseStyles';
+import ApplicationLayoutPropTypes from '../utils/propTypes';
+import Helpers from '../utils/helpers';
+
+import UtilityMenuWrapper from './_UtilityMenuWrapper';
+
+import styles from './ApplicationMenu.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The AppDelegate instance that will be propagated to the components presented within the NavigationLayout.
+   */
+  app: AppDelegate.propType,
+  /**
+   * The element to be placed within the fill flex styled content area.
```

Comment:
We should probably beef up this description to indicate this is passed from the user-defined configuration that we wrapped